libs/base/buffer_vector.hpp

```diff
@@ -25,22 +25,22 @@ class buffer_vector
     USE_DYNAMIC = N + 1
   };
   // TODO (@gmoryes) consider std::aligned_storage
-  T m_static[N];
+  T m_static[N]{};
```

Comment:
Is it really necessary? Can regress basic (buffer) usage.

Reply:
I had a warning that field is not initialized.
Why regress?